src/web/jcc/client/components/CommandControl.tsx

```diff
@@ -1958,11 +1958,17 @@ export default class CommandControl extends React.Component {
 	// Transferring Mission to GUI (End)
 	//
 
-	// 
-	// Bot Edit Mode (Start)
-	//
+	/**
```

Comment:
Can we update this comment to match the documentation: https://github.com/jaiarobotics/jaiabot/blob/1.y/src/doc/markdown/page110_coding_standards.md